Add Description meta (#7)
vippsas/vipps-magento-pwa-studio: README.md

![Vipps](./docs/images/vipps.png) *Available for Vipps.*

> ![MobilePay](./docs/images/mp.png) *Available for MobilePay in Finland. Expected availability in Denmark is Q1 2024.*

Author: ```suggestion
![MobilePay](./docs/images/mp.png) *Available for MobilePay in Finland and Denmark in Q1 2024.*
```
